clients/cloud/python/README.md

```diff
@@ -17,6 +17,24 @@ sasl.username=<api-key-id>
 sasl.password=<secret-access-key>
 ```
 
+## Configure SSL trust store
+
+Depending on your operating system or Linux distro you may need to take extra steps to set up the SSL CA root certificates.
+
+### CentOS
+
+```bash
+$ sudo yum reinstall ca-certificates
+```
+
+Add the following property to the config dict in the `Producer` and `AdminClient` objects in `producer.py`:
```

Comment:
".. objects in `producer.py` and `consumer.py`:"

Reply:
Fixed in https://github.com/confluentinc/examples/pull/144/commits/94393dbf11239b7c787739180ba19e8eb388a3f8